app/controllers/shipit/stats_controller.rb

```diff
@@ -0,0 +1,54 @@
+module Shipit
+  class StatsController < ShipitController
+    def show
+      @stats = {
+        average: 0,
+        min: 0,
+        max: 0,
+        median: 0,
+        success: 0
+      }
+      @diffs = {}
+
+      @previous_deploys = stack.deploys.where("created_at BETWEEN ? AND ?", 14.days.ago, 7.days.ago)
```

Comment:
You can use `where(created_at: 14.days.ago..7.days.ago)` for better cross DB compatibility.

Also naming that scope something like `previous_week` would make the code a bit nicer.